xml/FrameworksIndex/netframework-3.5.xml

```diff
@@ -2358,6 +2358,7 @@
       <Member Id="F:Microsoft.Build.Utilities.TargetDotNetFrameworkVersion.Version30" />
       <Member Id="F:Microsoft.Build.Utilities.TargetDotNetFrameworkVersion.Version35" />
       <Member Id="F:Microsoft.Build.Utilities.TargetDotNetFrameworkVersion.VersionLatest" />
+      <Member Id="F:Microsoft.Build.Utilities.TargetDotNetFrameworkVersion.VersionLatest" />
```

Comment:
@joelmartinez this new version of the CI is adding duplicate entries?

Reply:
@mairaw hmm, really strange … however the next version of mdoc will included [a change](https://github.com/mono/api-doc-tools/pull/386) that I've just made, that should definitely 100% avoid this going forward.